lib/Resque/Job.php

```diff
@@ -230,7 +230,12 @@ public function fail($exception)
 			$this->worker,
 			$this->queue
 		);
-		Resque_Stat::incr('failed');
+
+        if (Resque_Failure::getBackend() == 'Resque_Failure_RedisMultipleQueues') {
+            Resque_Stat::incr($this->queue . '_failed');
+        } else {
+            Resque_Stat::incr('failed');
+        }
```

Comment:
To be totally honest, I don't think resque_web even uses the counters and instead does all its work through an lrange instead (will have to check)

The reasoning behind having it like this is that it should work for those people who don't use multiple failure queues and we do want to push this back upstream to chrisboultons repo.

It might be better to do it inside the __construct methods of Resque_Failure_RedisMultipleQueues and Resque_Failure_Redis

thoughts?